code/math/vecmat.cpp

```diff
@@ -2954,6 +2954,46 @@ void vm_interpolate_angles_quick(angles *dest0, angles *src0, angles *src1, floa
 	dest0->b = src0->b + (arc_measures.b * interp_perc);
 }
 
+// Interpolate between two matrices, using t as a percentage progress between them.
```

Comment:
Since percentage generally means 0..100, I would either explicitly say 0..1, or else say "fraction" instead of "percentage"